xl-v1-object.go

```diff
@@ -154,10 +154,16 @@ func (xl xlObjects) getObjectInfo(bucket, object string) (objInfo ObjectInfo, er
 	return objInfo, nil
 }
 
```

Comment:
The comment is preserved. Since the definition of the function has moved relative to its original location in the file, it appears to be removed in the diff.